scss/application.scss

```diff
@@ -0,0 +1,142 @@
+
+p{
+
+  text-align: center;
+
+}
+
+.navbar-text {
+
+  font-size: 200%;
+  margin: 35px 20px 0 0;
+
+}
+
+.img-navbar{
+
+  margin: 0 0 0 20px;
+
+}
+
+.img_tech_logo{
+
+  height: 100px;
+  width: 100px;
+  margin: 0 0 0 60px;
+
+}
+
+
+.navbar{
+
+  height: 90px;
+  -webkit-box-shadow:0px 1px 1px;
+  -moz-box-shadow:0px 1px 1px;
+  box-shadow: 0px 15px 20px rgba(0,0,0,.1);
+  display: flex;
+  justify-content: space-between;
+
```

Comment:
The navbar should be steaky, that is, place it on top and it will stay there when you scroll :smile: